### PR TITLE
Add information about comics

### PR DESCRIPTION
**Pull request by MinecraftPlaye**
Add documentation about which comic metadata providers are currently
supported. This includes how to best name the comics that a future
online metadata provider can find the correct comic for most use cases.

The naming scheme is inspired by "comictagger" but simpliefies how
comic volumes are handled. The reason for this is that in the manga
world, an issue is referred to as a volume and ComicVine refers to a
series as a volume.

Instead of having to define what a volume is, the issue and series names
as well as the release year are used in the name. This should allow
a potential online metadata provider to find most comics.

Depends on jellyfin/jellyfin-plugin-bookshelf#38 and jellyfin/jellyfin-plugin-bookshelf#34
Might depend on  jellyfin/jellyfin-web#3034 as this PR adds support for saving the current position in the comic

### CODE REVIEW

**Comment by MinecraftPlaye** [comment]
Just wanted to note that all pull requests this PR depends on have been merged. ;)

File: general/server/media/comics.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+└── Comic (2008)
+    ├── ComicInfo.xml
+    └── Comic #001 (2008).cbr
+```
+
+Placing comics into a subfolder allows the placement of a `ComicInfo.xml` file for metadata parsing. Currently, Jellyfin does not support this for a series as a whole, but only for individual comics.
+
+## Local Metadata for "Comic Book Archive's"
+
+Should you're files not correspond to either `.cbz` or `.cbr` files, please refer to the [section on Books](xref:server-media-books) to see what is supported.
```
Comment (Shadowghost) [inline general/server/media/comics.md:37]:
```suggestion
Should your files not correspond to either `.cbz` or `.cbr` files, please refer to the [section on books](xref:server-media-books) to see what is supported.
```

File: general/server/media/comics.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+uid: server-media-comics
+title: Comics
+---
+
+# Comics / Mangas
+
+Comics and Mangas, from now on referred to as Comics, should usually be in the library root directory or in a subfolder for the individual comics. The subfolders allow for organization of metadata and images.
```
Comment (Shadowghost) [inline general/server/media/comics.md:8]:
```suggestion
Comics and mangas, from now on referred to as comics, should usually be in the library root directory or in a subfolder for the individual comics. The subfolders allow for organization of metadata and images.
```

File: general/server/media/comics.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Should you're files not correspond to either `.cbz` or `.cbr` files, please refer to the [section on Books](xref:server-media-books) to see what is supported.
+
+The following metadata formats are supported:
+
+- ComicInfo (from ComicRack)
+- ComicBookInfo (from ComicBookLover)
+
+> [!Note]
+> However, their are some limitations regarding the support of the ComicInfo format. `ComicInfo.xml` files can be parsed when they are placed in the same folder as the comic itself or when they are bundled within a `.cbz` file. `ComicInfo.xml` files bundled with any other archive format are not supported.
```
[the remark this answers is not in the document]
Comment (MinecraftPlaye) [reply general/server/media/comics.md]: Resolved in the latest commit ;)

**Review by Shadowghost — APPROVED** [review]

File: general/server/media/comics.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+# Comics / Mangas
+
+Comics and mangas, from now on referred to as comics, should usually be in the library root directory or in a subfolder for the individual comics. The subfolders allow for organization of metadata and images.
+
+For best results, it is recommended to name the files similar to the following schema where the volume number and issue number are used **interchangeable**.
+
+```txt
+Series_Name #IssueNumber (of Count) (PubYear).ext
```
Comment (crobibero) [inline general/server/media/comics.md:13]: Would this look something like the following?

```txt
Comics
├── Plastic Man #002 (of 1000) (1944).cbz
```

Comment (MinecraftPlaye) [reply general/server/media/comics.md:13]: For example. Another example would be `Attack on Titan #003 (2012).cbz` where I left out the `(of Count)` part as the series was still on-going when I named them. It's basically one of the name schemes from comictagger as I don't think it's necessary to re-invent the wheel here. ;)

**Pull request merged**